R/utils.R

```diff
@@ -200,27 +200,128 @@ getRasterLayerNames <- function(startDateTime, endDateTime, by, variables,
   return(layerNames)
 }
 
-#' Get all active FMI weather stations.
+# function fmi_station()
+
+# Reading the local version included within the package is separated into its
+# own function, since it's also used for tests.
+.fmi_stations_local <- function() {
+  system.file("extdata", "fmi_stations.csv", package="fmi") %>%
+    utils::read.table(header=TRUE, sep=";", as.is=TRUE) %>%
```

Comment:
True. Although for the odd cases (3 at the moment) where a station belongs to multiple groups, I now used `, ` as the separator, i.e. "Radio soundings, Weather stations". (I think that gives the table a "human-readable" feel to it, compared to a `;` separator.) But the use of `,` and `;` could be reversed, `quote=TRUE` used, or file saved as an `.rds`.